tests/python/unittest/test_contrib_gluon_data_vision.py

```diff
@@ -0,0 +1,145 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import mxnet as mx
+import numpy as np
+import scipy.ndimage
+from mxnet.test_utils import *
+from common import assertRaises, with_seed
+import shutil
+import tempfile
+import unittest
+
+def _get_data(url, dirname):
+    import os, tarfile
+    download(url, dirname=dirname, overwrite=False)
+    fname = os.path.join(dirname, url.split('/')[-1])
+    tar = tarfile.open(fname)
+    source_images = [os.path.join(dirname, x.name) for x in tar.getmembers() if x.isfile()]
+    if len(source_images) < 1 or not os.path.isfile(source_images[0]):
+        # skip extracting if exists
+        tar.extractall(path=dirname)
+    tar.close()
+    return source_images
+
+def _generate_objects():
+    num = np.random.randint(1, 10)
+    xy = np.random.rand(num, 2)
+    wh = np.random.rand(num, 2) / 2
+    left = (xy[:, 0] - wh[:, 0])[:, np.newaxis]
+    right = (xy[:, 0] + wh[:, 0])[:, np.newaxis]
+    top = (xy[:, 1] - wh[:, 1])[:, np.newaxis]
+    bot = (xy[:, 1] + wh[:, 1])[:, np.newaxis]
+    boxes = np.maximum(0., np.minimum(1., np.hstack((left, top, right, bot))))
+    cid = np.random.randint(0, 20, size=num)
+    label = np.hstack((cid[:, np.newaxis], boxes)).ravel().tolist()
+    return [2, 5] + label
+
+
+class TestImage(unittest.TestCase):
```

Comment:
Did you check this is completely covered by pytest's unittest support https://docs.pytest.org/en/latest/unittest.html ?

Reply:
Yes, it's modified to accomodate the transition from nose to pytest